### PR TITLE
Some typo fixes

### PR DESCRIPTION
**Pull request by arcalinea**
Found some typos while I was reading the paper and took the liberty of fixing them. These are all small spelling or grammar corrections.

### CODE REVIEW

File: README.mediawiki [hunk 1]
```diff
@@ -505,7 +505,7 @@ It is no coincidence that the battlefield of modern democracies is disputed in t
...
+The building blocks of institutions consist of facts that define agreements. But the kind of facts agreements contain are of a very specific type: Institutions are not built with objective facts that are scientific, measurable and independent from human judgment; but rather inter-subjective facts that build the social world within a community setting the relations of property and rights. For example, the notion that every red can of soda belongs to the Coca Cola Corporation is not objective but an inter-subjective fact agreed upon by all members of society acknowledging the intellectual property rights that a company has over its product. In this way, institutional reality helps scale economic relations and reduces the information required for organizations to transact.
```
Comment (domi41) [inline README.mediawiki]: fact agreed upon **by** all members

Wonder why github fails to highlight the added word…

File: README.mediawiki [hunk 2]
```diff
@@ -322,11 +322,11 @@ In October 2016, following a shocking, almost universally unexpected No to peace
...
+When Claude Shannon wrote his [http://math.harvard.edu/~ctm/home/text/others/shannon/entropy/entropy.pdf foundational 1948 paper on Information Theory], he was able to demonstrate how circuits can perform logic functions by expressing a binary state of 1 or 0 (<code>true</code> or <code>false</code> states). Since then, digital technology shaped the dynamics of all kinds of information systems. With this in mind we focused on building an efficient design for a governance machine able to operate with blockchains that keeps its human operators as sovereign rulers by means of the vote. In the same way bits move in computers signalling a <code>true</code> or <code>false</code> state, ''votes'' signal a boolean value for institutional decisions to be recorded under smart contracts.
```
Comment (domi41) [inline README.mediawiki]: 1 and 0 => 1 or 0

File: README.mediawiki [hunk 3]
```diff
@@ -188,7 +188,7 @@ Additionally, due to the risk that coercion through physical violence or threats
...
...rk led by researchers Hosp & Vora, an [https://pdfs.semanticscholar.org/24d5/5c866a7317dae11d37518b312ee460bc33d3.pdf Information Theory approach was taken to model voting systems] leading to the conclusion that a natural tension exists with a system aiming for ''perfect integrity'', ''perfect ballot secrecy'' and ''perfect tally verifiability''. All three cannot be simultaneously achieved when an adversary is computationally unbounded, able to brute force a system if unlimited time or memory are available. For this reason we consider it indispensable to implement digital democracies using blockchains. With network effects already in place, blockchains are able to verify transaction integrity and prevent token double-spending. Bitcoin’s [https://en.wikipedia.org/wiki/Proof-of-work_system ''proof of work''] model achieves this by rewarding computational capacity verifying transaction blocks (what is often referred as ''mining''), leading to a network “300 times more powerful than Google’s resources” according to pioneer [http://twitter.com/balajis Balaji Srinivasan]. For this reason, our design is based on tokens within a blockchain network operating as ''political cryptocurrency''.
```
Comment (domi41) [inline README.mediawiki]: we consider **it** indispensable

File: README.mediawiki [hunk 4]
```diff
@@ -378,7 +378,7 @@ Templates defining common practices for specific kinds of organizations are enco
 
 ====2.5.2 <code>Members</code>====
 
-Every Organization has members that get the right to vote on the decisions of the organization. Membership criteria is defined in the ''constitutional smart contract'' and is key for the trust on any democratic environment. Among the most common ways to subvert an election is the manipulation of voter registry. Securing this aspect with cryptographic means as well as an approval protocol is critical. Once a member is approved within an organization, he or she gets a specific amount of ''votes'' to be used for its governance.
+Every Organization has members that get the right to vote on the decisions of the organization. Membership criteria is defined in the ''constitutional smart contract'' and is key for the trust in any democratic environment. Among the most common ways to subvert an election is the manipulation of voter registry. Securing this aspect with cryptographic means as well as an approval protocol is critical. Once a member is approved within an organization, he or she gets a specific amount of ''votes'' to be used for its governance.
```
Comment (domi41) [inline README.mediawiki]: trust **in** any democratic environment

File: README.mediawiki [hunk 5]
```diff
@@ -438,7 +438,7 @@ Every <code>Organization</code> can have 1 or more cryptocurrency addresses to f
...
 
-* '''Percentage for <code>Issues</code>''': <code>Members</code> seeking to use resources from the <code>Organization</code> can request them by attaching a <code>Budget</code> to an <code>Issue</code>. A <code>Member</code> can request to used funds from a pool specifically reserved for this. If the final tally of a decision reaches a certain value (<code>true</code> or <code>false</code>), it can then enforce the final decision by unlocking coins or triggering a transaction sending the requested assets to a specific address.
+* '''Percentage for <code>Issues</code>''': <code>Members</code> seeking to use resources from the <code>Organization</code> can request them by attaching a <code>Budget</code> to an <code>Issue</code>. A <code>Member</code> can request to use funds from a pool specifically reserved for this. If the final tally of a decision reaches a certain value (<code>true</code> or <code>false</code>), it can then enforce the final decision by unlocking coins or triggering a transaction sending the requested assets to a specific address.
```
Comment (domi41) [inline README.mediawiki]: can request to **use** funds

**Comment by medied** [comment]
Thanks @arcalinea for catching and fixing these typos 👍 . And thanks @domi41 for highlighting them some of them too!

**Pull request merged**